akka-stream-tests/src/test/scala/akka/stream/io/DeprecatedTlsSpec.scala

```diff
@@ -516,7 +516,7 @@ class DeprecatedTlsSpec extends StreamSpec(DeprecatedTlsSpec.configOverrides) wi
 
         Source.single(SendBytes(ByteString.empty)).via(flow).runWith(Sink.ignore)
       }
-      Await.result(run("akka-remote"), 3.seconds) // CN=akka-remote
+      Await.result(run("pekko-remote"), 3.seconds) // CN=pekko-remote
```

Comment:
The CN in the trust store is `akka-remote`, probably best to keep it akka-remote for now and open a separate issue to create new keystore/truststore for pekko.

Reply:
I've reverted the changes in these 2 tests